packages/labs/gen-wrapper-react/goldens/test-element-a/src/element-slots.ts

```diff
@@ -3,9 +3,9 @@ import {createComponent} from '@lit-labs/react';
 
 import {ElementSlots as ElementSlotsElement} from '@lit-internal/test-element-a/element-slots.js';
 
-export const ElementSlots = createComponent(
-  React,
-  'element-slots',
-  ElementSlotsElement,
-  {}
-);
+export const ElementSlots = createComponent({
+  react: React,
+  tagName: 'element-slots',
+  elementClass: ElementSlotsElement,
+  events: {},
```

Comment:
It looks like events can be `undefined` or `null` (and in the implementation we do a defensive `?? {}`. Is it worth the complexity to not have this field emitted in the empty case?

Non blocking.

Reply:
I'd say it's pretty harmless to emit empty object instead of trying to remove the option entirely.